scripts.js

```diff
@@ -9,7 +9,11 @@ const colors = [
   'orange'
 ];
 
-const allColors = [...colors, ...colors];
+const totalArrayLength = 10;
+
+const oneSetOfColors = colors.slice(0, totalArrayLength / 2);
```

Comment:
There is still some kind of duplication going on. Try not to overcomplicate things, usually, the simplest solution is better 😉